### ISSUE TITLE
Provide instructions for administrators

### ISSUE THREAD

**Issue by timcu** [timcu/sas-select#41]
For now this could go in README.md. Should include:
1. How to do a search and then view entitlement in packs.
2. How to load the latest excel data file. https://sas-select.triptera.com.au/fetch-data
3. How to add new special cases like 28300 where packs are split. Currently requires programming change.

**Comment by chayawit** [comment]
Off-topic but the link doesn't work. Seems like The site has to be http only.

**Comment by timcu** [comment]
> Off-topic but the link doesn't work. Seems like The site has to be http only.

I have enabled https on sas-select.triptera.com.au

**Comment by chayawit** [comment]
## Instructions for administrators
### How to do a search
Type company code and/or brand name, then press enter key or click search button.

The search query can be partial, e.g., querying company code `23` would also yield products with company code `123` or `1234`. 

However, if products with company code `23` exist in the database, `123` and `1234` won't be shown in the results.

In either case, maximum search results are 20.

### How to view entitlement in packs.
After searching, click the `view` button to see all info of the product including pack entitlement, which is calculated from pack size and medicare entitlement.

### How to load the latest excel data file
Then name of excel file in used is always shown in the footer of every page.

Typing `/fetch-data` at the end of the home page URL tells the application to look for new excel data file and put it into the database.

`https://sas-select.triptera.com.au/fetch-data`



### How to add new special cases where packs are split

Most product packs aren't allowed to be split. Special cases are stored in `special_entitlements` dict in `__init__.py`, which requires programming change to add any new special cases.

Each key in the dict is `(company_code, brand_name)` tuple where its value is another dict that stores number of packs and frequency.

The `special_entitlements` dict initially has only one entry:
 
`('28300', 'Omnigon Bbraun Urimed Bag 2L'): {'packs': 0.5, 'frequency': 'month'}`

28300 is the company code of this product. Omnigon Bbraun Urimed Bag 2L is the brand name.

Taking this as an example, one can add a new entry by adding a `,` and the new entry.

**Comment by chayawit** [comment]
Please review. I can also add some screenshots but then we would need a new directory to keep them.

**Comment by timcu** [comment]
Looks good. A couple of suggestions:

- Add an initial section: "Running the web app" https://sas-select.triptera.com.au
- How to do a search: Give real examples (eg)
  - Company code: 28300, Brand name: Hollister 
  - Company code: 28, Brand Omnigon
  - Brand name: Omnigon Bbraun Urimed Bag 2L
  - Company code: CFL28
- View entitlements section: `view` button should be `view` link

I don't think we need screenshots yet. The section "How to search" could probably be added to the bottom of the "Search" page to assist all users.

**Comment by chayawit** [comment]
> * Add an initial section: "Running the web app" https://sas-select.triptera.com.au

What do you mean by running? Is it opening the website with a web browser? I thought you meant deploying but I saw deploying info on the building.md page.

**Comment by timcu** [comment]
Yes, running the web app is opening the website with a web browser. We need to provide some guidance to someone who has not seen the site before.

**Comment by chayawit** [comment]
### Running the web app
Open https://sas-select.triptera.com.au/ with a web browser.

### How to do a search
Type company code and/or brand name, then press enter key or click search button.

To ensure the most accurate result, please type the full company code and brand name.

For example, try Company code: 28300 and Brand name: Hollister.

In the cases where you are not sure about the Company code or brand name, partial Company code and Brand name and be entered.

For instance, each product result for the query of Company code: 28 and Brand name: Omnigon would have “Omnigon” its brand name and also “28” in its Company code. These could include:
- Company code: 4**28**15, Brand Name, Omnigon Almarys Flexima
- Company code: 4**28**17, Brand Name, Omnigon Flexima

Please note that if product(s) with the exact company code is found in the database, other company codes that contain the search query will not be in the result list.

For example, a user might search Company Code :3184 expecting the product with Company Code 41**3184**, Brand Name: ConvaTec Sur-Fit Natura. But the only result will be Company Code: 3184, Brand Name: Hollister Micro. In this case, if the user add the Brand Name: ConvaTec into the search query, the result will be as expected.

In either case, maximum search results are 20.
